pcdet/models/roi_heads/pvrcnn_head.py

```diff
@@ -186,19 +189,15 @@ def forward(self, batch_dict, test_only=False,use_gtboxes=False):
         if use_gtboxes == True:
             # batch_dict['pooled_features_gt'] = pooled_features
             batch_size_rcnn = pooled_features.shape[0]
-            pooled_features_copy = pooled_features.clone()
-            start_epoch = self.model_cfg['ROI_HEAD'].get('INSTANCE_CONTRASTIVE_LOSS_START_EPOCH', 0)
-            stop_epoch = self.model_cfg['ROI_HEAD'].get('INSTANCE_CONTRASTIVE_LOSS_STOP_EPOCH', 60)
-            
+            start_epoch = self.model_cfg['INSTANCE_CONTRASTIVE_LOSS_START_EPOCH']
+            stop_epoch = self.model_cfg['INSTANCE_CONTRASTIVE_LOSS_STOP_EPOCH']
             if self.model_cfg.ENABLE_INSTANCE_SUP_LOSS==True and start_epoch<=batch_dict['cur_epoch']<stop_epoch: # normalize embedding and produce projected representation only when instance_sup_loss true.
-
                 if self.model_cfg['NORMALIZATION']:
-                    pooled_features_copy = F.normalize(pooled_features, dim = -1)
-                    if self.model_cfg['PROJECTOR']:
-                        shared_features_gt = self.projector_fc_layer(pooled_features_copy.view(batch_size_rcnn, -1, 1))
-                else:
-                    shared_features_gt = self.projector_fc_layer(pooled_features.view(batch_size_rcnn, -1, 1))
-                batch_dict['shared_features_gt'] = shared_features_gt
+                    # pooled_features dim : [GT_boxes,27648]
```

Comment:
Can you explain this part here because in the next commit you also changed it? @advait